lib/ui/src/components/sidebar/Menu.tsx

```diff
@@ -45,9 +45,20 @@ export const MenuButton = styled(Button)<MenuButtonProps>(({ highlighted, theme
   position: 'relative',
   overflow: 'visible',
   padding: 7,
+  transition: none; //prevents button border from flashing when focused/blurred
```

Comment:
This line is creating errors when I try to run it locally

```SyntaxError: storybook/lib/ui/src/components/sidebar/Menu.tsx: Unexpected token, expected "," (48:18)```

When I try to change it to a `,` it tells me that `none` is not defined.

Reply:
what if you change it to:

```
transition: 'none',
```